src/components/button/index.tsx

```diff
@@ -124,6 +124,17 @@ class Button extends PureComponent<Props, ButtonState> {
     return color;
   }
 
+  getIconColor(): string | undefined {
+    const {disabled} = this.props;
+    let tintColor = this.getLabelColor();
```

Comment:
That ignores the tintColor passed in iconStyle and iconProps, no?

Reply:
no, passing iconStyle is always prioritized and it's include in the `getIconStyle` method

Reply:
Ok. I thought it was the props.iconStyle that is passed, I missed line #306 (which means the naming is confusing :) Maybe change to _iconStyle?)

Reply:
Wait. You mean it is prioritized in the Icon component because you now override the style with props.style, right?